src/config.ts

```diff
@@ -59,3 +100,88 @@ function createManifestTransform(base: string): import('workbox-build').Manifest
     return { manifest: entries, warnings: [] }
   }
 }
+
+function createSSRHandler(route: string): RuntimeCaching {
+  return {
+    urlPattern: route,
+    handler: 'NetworkOnly',
+  }
+}
+
+function createSSRHandlerFunction(
+  cache: boolean,
+  cacheName: string,
+  offlinePage: string,
+  regex: RegExp,
+  dynamic: boolean,
+): RuntimeCaching {
+  return cache && !dynamic
+    // eslint-disable-next-line no-eval
+    ? eval(`() => ({
+    urlPattern: ({ url, sameOrigin }) => sameOrigin && url.pathname.match(${regex}),
+    handler: 'NetworkFirst',
+    options: {
+      cacheName: ${JSON.stringify(cacheName)},
+      cacheableResponse: {
+        statuses: [200]
+      },
+      matchOptions: {
+        ignoreVary: true,
+        ignoreSearch: true
+      },
+      plugins: [{
+        handlerDidError: async () => Response.redirect(${JSON.stringify(offlinePage)}, 302),
+        cacheWillUpdate: async ({ response }) => response.status === 200 ? response : null
```

Comment:
prevent error here (response can be null/undefined when offline)?
`cacheWillUpdate: async ({ response }) => response?.status === 200 ? response : null`